src/main/java/io/debezium/connector/yugabytedb/YugabyteDBErrorHandler.java

```diff
@@ -16,14 +18,22 @@
  * @author Suranjan Kumar, Vaibhav Kushwaha
  */
 public class YugabyteDBErrorHandler extends ErrorHandler {
+    private final static Logger LOGGER = LoggerFactory.getLogger(YugabyteDBErrorHandler.class);
 
     public YugabyteDBErrorHandler(YugabyteDBConnectorConfig connectorConfig, ChangeEventQueue<?> queue) {
         super(YugabyteDBConnector.class, connectorConfig, queue);
     }
 
     @Override
     protected boolean isRetriable(Throwable throwable) {
+        LOGGER.info("Checking if a retry is possible for retriable exception: {}", throwable);
+
+        if (throwable.getMessage() == null) {
+            LOGGER.warn("Exception message received is null");
```

Comment:
We should log the throwable if it is not being logged.

Reply:
It is being logged in line number 29.